### DISCUSSION TITLE
Getting current cursor position

### DISCUSSION THREAD

**Discussion by avdept** [krzyzanowskim/STTextView#4] [Questions]
Hey @krzyzanowskim was wondering if you have any way to get current cursor position - row and column?

**Comment by krzyzanowskim** [comment]
I wonder whether the STTextView.didChangeSelectionNotification is insufficient for that purpose. It's modeled after how NSTextView works. The notification is fired for every selection change, and since the caret position is a selection, it is fired on every caret position change too.

The didChangeSelectionNotification notification is fired after change, meaning the handler can obtain the new position from the STTextView.
